app/src/org/commcare/gis/EntityMapboxActivity.kt

```diff
@@ -81,10 +84,17 @@ class EntityMapboxActivity : BaseMapboxActivity() {
     private lateinit var source: GeoJsonSource
     private var iconset = java.util.HashSet<String>()
     private lateinit var mapEntities: java.util.ArrayList<MapEntity>
+    private lateinit var viewBinding: ActivityEntityMapboxBinding
 
+    override fun getViewBinding(): ViewBinding {
+        if (!::viewBinding.isInitialized) {
+            viewBinding = ActivityEntityMapboxBinding.inflate(layoutInflater)
+        }
+        return viewBinding
+    }
 
-    override fun getMapLayout(): Int {
-        return R.layout.activity_entity_mapbox
+    override fun getMapView(): KujakuMapView {
```

Comment:
is this same as it's super method in `BaseMapboxActivity` ?

Reply:
Yes. Also, this method is abstract in `BaseMapboxActivity`